lib/preload.js

```diff
@@ -734,7 +745,7 @@ class Preloader extends EventEmitter {
 				baseUrl: this.balena.pine.API_URL,
 				url: '/auth/v1/token',
 				qs: {
-					service: this.balenaSettings.registry2Url,
+					service: this._registryUrl(images[0]),
```

Comment:
This will return a valid token even if registry2 isn't a reachable URL. Still good to use a valid endpoint though!